File: Easy/find_the_index_of_the_first_occurrence_in_a_string.go
package main

import (
	"fmt"
	"strings"
)

func strStr(haystack string, needle string) int {
	return strings.Index(haystack, needle)
}

func run_strStr() {
	fmt.Println(strStr("sadbutsad", "sad"))
	fmt.Println(strStr("leetcode", "leeto"))
}

File: Easy/find_the_index_of_the_first_occurrence_in_a_string.js
/**
 * @param {string} haystack
 * @param {string} needle
 * @return {number}
 */
var strStr = function(haystack, needle) {
    return String(haystack).search(needle)
};

console.log(strStr("sadbutsad", "sad"));
console.log(strStr("leetcode", "leeto"));